### PR TITLE
WR-1705 add Related Topics Image field, add Related Topics Card content type,…

### PR DESCRIPTION
**Pull request by rjkuyvenhoven**
… update Related Topics Block to handle new image field and content type

### CODE REVIEW

File: src/config/sync/core.entity_form_display.node.blog.default.yml [hunk 1]
```diff
@@ -18,7 +19,6 @@ dependencies:
     - content_moderation
     - datetime
     - media_library
-    - media_library_edit
```
Comment (infojunkie) [inline src/config/sync/core.entity_form_display.node.blog.default.yml:21]: Why is this removed from here and other form displays below? Did we lose the edit icon on media image fields?

File: src/config/sync/core.entity_form_display.node.related_topics_card.default.yml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      placeholder: ''
+    third_party_settings: {  }
+  field_related_topics_image:
+    type: media_library_widget
+    weight: 2
+    region: content
+    settings:
+      media_types: {  }
+      form_mode: media_library
+    third_party_settings: {  }
```
Comment (infojunkie) [inline src/config/sync/core.entity_form_display.node.related_topics_card.default.yml:59]: Make sure to show the media image edit icon here as well.

**Review by infojunkie — CHANGES_REQUESTED** [review]
Looks good! Few comments to answer please.

File: src/config/sync/core.entity_form_display.node.blog.default.yml (hunk 1 above)
Comment (rjkuyvenhoven) [reply src/config/sync/core.entity_form_display.node.blog.default.yml:21]: Hmmm, I don't know. Will investigate.

**Review by infojunkie — APPROVED** [review]
Looks like testing showed the edit icon is still there, so good to go!
